Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Active_Directory_Enrichment.yml

```diff
@@ -327,8 +328,696 @@ tasks:
+            operator: split
+          - operator: FirstArrayElement
+          - args:
+              limit: {}
+              replaceWith: {}
+              toReplace:
+                value:
+                  simple: CN=
+            operator: replace
+          - operator: uniq
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Active Directory Query v2
+      description: Retrieves detailed information about a user account. The user can be specified by name, email address, or as an Active Directory Distinguished Name (DN). If no filter is specified, all users are returned.
+      id: 56097741-3c3f-4aff-82e8-e2dbe755cf92
+      iscommand: true
+      name: Get User Manager Details
+      script: Active Directory Query v2|||ad-get-user
+      type: regular
+      version: -1
+    taskid: 56097741-3c3f-4aff-82e8-e2dbe755cf92
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": -310,
+          "y": 1060
+        }
+      }
+  "21":
+    continueonerrortype: ""
+    id: "21"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "23"
+      - "24"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 70bfa636-ba5c-4f8e-837c-8bd1ccadf737
+      iscommand: false
+      name: Set Fields
+      type: title
+      version: -1
+      description: ''
+    taskid: 70bfa636-ba5c-4f8e-837c-8bd1ccadf737
+    timertriggers: []
+    type: title
+    view: |-
+      {
+        "position": {
+          "x": -860,
+          "y": 1250
+        }
+      }
+  "22":
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              complex:
+                accessor: name
+                root: ActiveDirectory.Users
+                transformers:
+                - operator: uniq
+          operator: isNotEmpty
+          right:
+            value: {}
+      label: "yes"
+    continueonerrortype: ""
+    id: "22"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#default#':
+      - "2"
+      "yes":
+      - "19"
+      - "21"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Ownership Information exists?
+      id: c77f9e47-944a-4ba4-8e87-6cc125230f5f
+      iscommand: false
+      name: Ownership Information exists?
+      type: condition
+      version: -1
+    taskid: c77f9e47-944a-4ba4-8e87-6cc125230f5f
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": -310,
+          "y": 590
+        }
+      }
+  "23":
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              simple: ActiveDirectory.Users.mail
```

Comment:
I think this isn't looking at the right place in context.  Because we did the extend-context to ${manager-email}, we should be looking in there

Reply:
Great catch. Updated it.